### PR TITLE
add pygadm lib

### PR DESCRIPTION
**Pull request by 12rambau**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by 12rambau** [comment]
@conda-forge/help-python, ready for review!

File: recipes/pygadm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{% set name = "pygadm" %}
+{% set version = "0.0.0" %}
```
Comment (BastianZim) [inline recipes/pygadm/meta.yaml:2]: Is this correct?

File: recipes/pygadm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pygadm-{{ version }}.tar.gz
+  sha256: fa14ff5a134e2af2b9670da787c22e67a6cfb9c55552727a3f27c7ff355dad37
+
+build:
```
Comment (BastianZim) [inline recipes/pygadm/meta.yaml:12]: Please also add the entry_points: https://github.com/12rambau/pygadm/blob/2c682d3781817eaa724a43120b720c35a45697b7/pyproject.toml#L67

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just a couple of small things, then this is ready to go 🚀

File: recipes/pygadm/meta.yaml (hunk 1 above)
Comment (12rambau) [reply recipes/pygadm/meta.yaml:2]: not anymore ;-) let me do a quick update

File: recipes/pygadm/meta.yaml (hunk 2 above)
Comment (12rambau) [reply recipes/pygadm/meta.yaml:12]: that's for developers only, let me remove it from the lib.

Comment (12rambau) [reply recipes/pygadm/meta.yaml:12]: removed in v0.2.0

**Pull request merged**